### DISCUSSION TITLE
Missing qtxdg-tools in master branch

### DISCUSSION THREAD

**Discussion by WojtasJD** [lxqt/lxqt#2240] [Q&A] [answered]
Hi. I have a error while building LXQt 1.1.0 from sources with script to build in one step on Debian 11 (missing qtxdg-tools configuration files). If I switch to the "add-qtxdg-tools" branch earlier - the building process ends successfully.
Am I doing something wrong?

**Comment by tsujan** [comment, marked as answer]
> Am I doing something wrong?

I don't think so.

I merged https://github.com/lxqt/lxqt/pull/2217 right now. Please see if it fixes the issue.

**Reply by WojtasJD** [reply]
Thank you, it's working.
